pkg/quota/evaluator/core/pods.go

```diff
@@ -245,13 +324,51 @@ func PodUsageFunc(obj runtime.Object) (api.ResourceList, error) {
 	for i := range pod.Spec.Containers {
 		requests = quota.Add(requests, pod.Spec.Containers[i].Resources.Requests)
 		limits = quota.Add(limits, pod.Spec.Containers[i].Resources.Limits)
+		for _, volumeMount := range pod.Spec.Containers[i].VolumeMounts {
```

Comment:
Use a helper function to remove duplicated code for container and init container,